dspace-api/src/main/java/org/dspace/subscriptions/ContentGenerator.java

```diff
@@ -70,7 +78,7 @@ private String generateBodyMail(Context context, List<IndexableObject> indexable
         try {
             ByteArrayOutputStream out = new ByteArrayOutputStream();
             out.write("\n".getBytes(UTF_8));
-            if (indexableObjects.size() > 0) {
+            if (indexableObjects != null && !indexableObjects.isEmpty()) {
```

Comment:
`.isEmpty()` check should be unnecessary.  If the List is empty, the `for` loop should exit immediately.

Perhaps the `null` test should be the first thing done, returning `""` if true.